megamek/src/megamek/common/MechView.java

```diff
@@ -475,6 +472,41 @@ public MechView(final Entity entity, final boolean showDetail, final boolean use
             list.addItem(quirksList.toString());
             sFluff.add(list);
         }
+
+        Game game = entity.getGame();
+
+        if ((game == null) || game.getOptions().booleanOption(OptionsConstants.ADVANCED_STRATOPS_QUIRKS)) {
```

Comment:
What's the "real life" situation where the game is null? Do we actually want to default to displaying quirks in that situation?

Reply:
this is in mekhq and megameklab.   i would think you would want them displayed then.

Reply:
ok